docs/guides/force-token-refresh.mdx

```diff
@@ -0,0 +1,244 @@
+
+## Example
+
+Say you're building a user profile page. You want to allow the user to update their information, and also allow them to set and update custom information. To store information about a user that Clerk doesn't collect, you can use [metadata](/docs/users/metadata), which will get stored on the user's `User` object as `User.publicMetadata`.
+
+When using metadata, it's recommended to [store it in the user's session token](/docs/backend-requests/custom-session-token) to avoid making a network request to Clerk's Backend API when retrieving it.
+
+However, when retrieving the user's metadata from the session token, because the session token refreshes every 60 seconds, the metadata may not be up to date if the token hasn't refreshed yet. In this case, you'd want to force a session token refresh. This is when you could use `getToken({ skipCache: true })`.
+
+So why does this example use `user.reload()` instead? Because on this user's profile page, the metadata is not the only information the user can update. They can also update their first and last name. Therefore, you not only want to refresh the session token, which is where the metadata is retrieved from, but also the `User` object, which is where the first and last name is retrieved from. This is why `user.reload()` is the better option. If you were only retrieving the user's metadata, and not using the `User` object, you could use `getToken({ skipCache: true })` instead.
+
+**This example requires that the user's metadata has been configured to be stored in the session token under the `metadata` key.** To do so, follow the [guide on customizing your session token](/docs/backend-requests/custom-session-token).
+
+<Tabs items={["API route", "Client-side page"]}>
+  <Tab>
+    The API route that uses Clerk's [`updateUserMetadata()`](/docs/references/backend/user/update-user-metadata) method to update the user's metadata.
+
+    ```ts {{ filename: 'app/api/update-user-metadata/route.ts', collapsible: true }}
+    import { auth, clerkClient } from '@clerk/nextjs/server'
+    import { NextRequest, NextResponse } from 'next/server'
+
+    export async function POST(req: NextRequest) {
+      // Use `req.json()` to parse the request body
+      const metadata = await req.json()
+
+      // Use `auth()` to access the current user's ID
+      const { userId } = await auth()
+
+      // Protect the route by checking if the user is signed in
+      if (!userId) {
+        return Response.json({ error: 'Unauthorized' }, { status: 401 })
+      }
+
+      // Use `clerkClient()` to access the Clerk client
+      const client = await clerkClient()
+
+      // Use the Backend SDK's `updateUserMetadata()` method to update the user's metadata
+      await client.users.updateUserMetadata(userId, {
+        publicMetadata: {
+          ...metadata,
```

Comment:
I know this is just an example, but this would be really bad code to write. Any user could just dump the full works of shakespeare into this api route and probably break your app. It would probably be a better, safer example if you updated some specific piece of metadata here rather than just taking user input and piping it straight in